src/Homepage/Collections/CustomDropdown.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+
+import './_custom-dropdown.scss';
+
+
+export default class CustomDropdown extends Component {
+    onCollectionChange = (event) => {
+        this.props.onChange(event.target.value)
+    }
+
+    render() {
+        const {optionArray} = this.props;
+
+        return (
+            <div className={"custom-dropdown-wrapper"}>
+                <div className={"shadow"}> </div>
+                <select className={"select-collections"}
+                        onChange={this.onCollectionChange}>
+                    {optionArray.map(option =>
+                        <option key={option.id}
+                                value={option.id}>
+                                {option.title}
+                        </option>
+                    )}
+                </select>
+
+                {this.props.selectedOption}
+            </div>
+        );
+    }
+}
+// {
```

Comment:
Since you're using Git version control system, you don't need to keep legacy code with comment lines. Let's remove commented out codes.